src/blocks/ad-unit/index.js

```diff
@@ -24,12 +24,13 @@ export const icon = (
 		<Path d="M0 0h24v24H0z" fill="none"/>
 	</SVG>
 );
+console.log( getCategories() );
```

Comment:
Leftover debug code

Reply:
Fixed in https://github.com/Automattic/newspack-ads/pull/4/commits/1d7cb0e105536b10046fe20fc928b277ee55507a